### PR TITLE
[3007.x] Merge forward 3006.x into 3007.x

### PR DESCRIPTION
**Pull request by dwoz**
### What issues does this PR fix or reference?
Fixes

### Previous Behavior
Remove this section if not relevant

### New Behavior
Remove this section if not relevant

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [ ] Changelog - https://docs.saltproject.io/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### Commits signed with GPG?
Yes/No

Please review [Salt's Contributing Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html) for best practices, including the
[PR Guidelines](https://docs.saltproject.io/en/master/topics/development/pull_requests.html).

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

File: requirements/windows.txt [hunk 1]
```diff
@@ -3,3 +3,34 @@
 # If they are windows specific, place "; sys_platform == 'win32'" in front of the requirement.
 
 -r zeromq.txt
+# <<<<<<< HEAD
```
Comment (dmurphy18) [inline requirements/windows.txt:6]: got some need human to resolve conflicts here

**Review by dmurphy18 — CHANGES_REQUESTED** [review]
Just the human resolve needed

File: requirements/windows.txt (hunk 1 above)
Comment (dwoz) [reply requirements/windows.txt:6]: Left this here while I validated the dependencies after migrating them to `base.txt`. I will clean it up before merging anything.

Comment (dmurphy18) [reply requirements/windows.txt:6]: Wait I thought it was like green M&Ms on the rider, checking if we review :rofl:

**Pull request closed without merging**